tests/integration_tests/performance/test_net_tput.py

```diff
@@ -0,0 +1,290 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the network throughput overhead added by Firecracker."""
+
+import json
+import time
+import statistics
+from subprocess import run
+import pytest
+
+import host_tools.network as net_tools
+
+RUNS = 1
+
+IPERF = 'iperf3'
+ITERATION_DURATION = 1
+WINDOW_SIZE = 10
+
+# BW in Mbits.
+BANDWIDTHS = {
+    'udp': [100, 5000, 10000],
+    'tcp': [5000, 10000, 20000, 0]
+}
+
+# Packet size used for TCP.
+PACKET_SIZE = [128, 256, 1024, 1500]
+
+# From server to client and reversed.
+MODES = ['h2g', 'g2h']
+# Corresponding MODES for iperf options.
+MODES_OPTIONS = {
+    'h2g': '-R',
+    'g2h': ''
+}
```

Comment:
can we merge these variables ?

I'm not sure about the exact syntax, but 'm thinking about something like :

```
Modes: {
h2g: {
options: '-R'
}
...
```

Reply:
Sure, we can do this